consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/validation/RoundChangePayloadValidator.java

```diff
@@ -113,6 +117,18 @@ private boolean validateConsistencyOfPrepareCertificateMessages(
     return true;
   }
 
+  private boolean hasDuplicateAuthors(
+      final Collection<SignedData<PreparePayload>> preparePayloads) {
+    final long distinctAuthorCount =
```

Comment:
can return early with `true` if preparePayloads list is empty?

Reply:
Probably could return early, however there's probably not much of a performance gain, and adds lines of code which would then need to be tested, so would rather not ... but can be convinced if there's significant benefits.